src/Microsoft.Diagnostics.Monitoring.Egress/LoggingExtensions.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Extensions.Logging;
+using System;
+using System.Globalization;
+
+namespace Microsoft.Diagnostics.Monitoring.Egress
+{
+    internal static class EgressOptionsLogging
```

Comment:
>EgressOptionsLogging [](start = 26, length = 20)

For the default console logger for dotnet-monitor, it may be worthwhile to add scopes information so that we can correlate our textual logs with uploads. It would be good to see how ContainerInsights processes that information.